### ISSUE TITLE
pydantic requirement error

### ISSUE THREAD

**Issue by flaviojdz** [jpetrucciani/mypy-check#4]
How do I approach solving this error? is there a way to trigger a pip install for extra mypy requirements?

mypy 0.910
+ tee /tmp/mypy.out
+ mypy --show-column-numbers --hide-error-context '*.py'
setup.cfg:22:1: error: Error importing plugin "pydantic.mypy": No module named 'pydantic'
Found 1 error in 1 file (errors prevented further checking)
+ exit_code=2
+ python /github.py /tmp/mypy.out
+ exit 2

**Comment by jpetrucciani** [comment]
Do you have your requirements listed in a `requirements.txt` or something like that? Would an option to `pip install -r` a file be useful?

**Comment by virtuald** [comment]
Yes, that would be useful, otherwise mypy fails for anything that has dependencies on non-stdlib packages.

**Comment by jpetrucciani** [comment]
The latest release (or `@master`) should now support `requirements` and `requirements_file` as options!
```yml
- uses: jpetrucciani/mypy-check@master
  with:
    requirements_file: requirements.dev.txt
```
